feature/home/src/main/java/com/yapp/feature/home/component/FAQSection.kt

```diff
@@ -0,0 +1,89 @@
+package com.yapp.feature.home.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.yapp.core.designsystem.extension.yappClickable
+import com.yapp.core.designsystem.theme.YappTheme
+import com.yapp.core.designsystem.R as coreDesignR
+
+@Composable
+internal fun FAQSection(
+    modifier: Modifier = Modifier,
+    clickBasicRule: () -> Unit,
+    clickCurriculum: () -> Unit,
+) {
+    Column(modifier = modifier.fillMaxWidth()) {
+        Text(text = "혹시... 처음이 YAPP?", style = YappTheme.typography.headline1Bold)
+        Spacer(Modifier.height(8.dp))
+        Faq(
+            clickBasicRule = clickBasicRule,
+            clickCurriculum = clickCurriculum
+        )
+    }
+}
+
+@Preview
+@Composable
+private fun FaQSectionPreview() {
+    YappTheme { FAQSection(clickCurriculum = {}, clickBasicRule = {}) }
+}
+
+@Composable
+private fun Faq(
+    clickBasicRule: () -> Unit,
+    clickCurriculum: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .clip(RoundedCornerShape(16.dp))
+            .background(color = YappTheme.colorScheme.staticWhite)
+            .padding(horizontal = 16.dp, vertical = 8.dp)
+    ) {
+        Row(
+            modifier = Modifier
+                .padding(vertical = 12.dp)
+                .fillMaxWidth()
+                .yappClickable(onClick = { clickBasicRule() }),
+            horizontalArrangement = Arrangement.spacedBy(4.dp),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Image(painter = painterResource(coreDesignR.drawable.yappo_wink), contentDescription = null)
+            Text(text = "YAPP 기본 규칙")
```

Comment:
R:
```suggestion
            Text(
                text = "YAPP 기본 규칙",
                style = YappTheme.typography.body1NormalRegular,
                color = YappTheme.colorScheme.labelNormal
            )
```

R: YAPP 기본 규칙이랑 N기 커리큘럼 글꼴 스타일 적용해주세요

Reply:
[344aabb](https://github.com/YAPP-admin/yappu-world-android/pull/226/commits/344aabb0eb53ae17031dabdaeb8f06763b84aa57)

처리완료 했어요